src/software/simulated_tests/defense_play_test.cpp

```diff
@@ -16,7 +16,7 @@ TEST_F(DefensePlayTest, test_defense_play)
 {
     setBallState(BallState(Point(0.9, 2.85), Vector(0, 0)));
     addFriendlyRobots(TestUtil::createStationaryRobotStatesWithId(
-        {Point(-3, 2.5), Point(-3, 1.5), Point(-3, 0.5), Point(-3, -0.5), Point(-3, -1.5),
+        {Point(-4.5, 0), Point(-3, 1.5), Point(-3, 0.5), Point(-3, -0.5), Point(-3, -1.5),
```

Comment:
Why did you change the first position from (-3, 2.5) to (-4.5, 0) for all of your test cases? Otherwise LGTM :)

Reply:
The first position refers to the goalkeepers position at the start of the test. `(-3, 2.5)` is outside the `defender_crease`, which is essentially an inflated version of our defense rectangle. For all tests that used the `crease_defender_tactic`, this caused the warning:

`WARNING [crease_defender_tactic.cpp->calculateDesiredState:114] Could not draw a line from the goalie to the ball through the defender crease` 

Changing the goalkeepers position to `(-4.5,0)` means the goalkeeper starts inside the `defender_crease`, so we dont get that warning anymore.